scripts/create-package.ts

```diff
@@ -195,9 +195,6 @@ const DEFAULT_COMPONENT_NAME = 'Component'
    * Add package to dependencies.
    */
   if (dependencies) {
-    /*dependencies.forEach((dependency) =>
-      exec(`yarn lerna add ${dependency} --exact --scope=${packageName}`),
-    )*/
     exec(`yarn workspace ${packageName} add ${dependencies.join(' ')} --exact`)
```

Comment:
I assume this is equivalent to what we had before but in a single `exec`, right?

Reply:
Yes. That's right